DataProcessor/DataUnpacker.cpp

```diff
@@ -164,6 +164,7 @@ void DataUnpacker::eng_dash_connection(bool state) {
 }
 
 bool DataUnpacker::checkRestartEnable() {
-    return battery_eStop || driver_eStop || external_eStop || imd_status || !door || crash || mcu_check || restart_enable;
+    //TODO Change mcu_hv_en to 1
+    return !restart_enable? !mcu_hv_en : false || driver_eStop || external_eStop || imd_status || door || crash || mcu_check || restart_enable || discharge_enable;
```

Comment:
```suggestion
    return (!restart_enable ? !mcu_hv_en : false) || driver_eStop || external_eStop || imd_status || door || crash || mcu_check || discharge_enable || restart_enable;
```
nit: Can we add parentheses and move `discharge_enable` for readability? Also, we don't need the mcu_hv_en TODO comment if that's already implemented